electron-app/src/components/settings/api-keys/ExternalServices.vue

```diff
@@ -11,6 +21,7 @@
           v-model="etherscanKey"
           class="external-services__etherscan-key"
           title="Etherscan"
+          description="Required for any Ethereum blockchain balances or transactions. Rotki uses etherscan to obtain basic information about ethereum blockchain addresses and transactions."
```

Comment:
Required for any Ethereum blockchain balances or transactions. Rotki uses etherscan to query ethereum blockchain data if you are not using your own ethereum node. If you are it's still needed but only for historical data.